v/vipnytt__sitemapparser/vipnytt__sitemapparser_UBI_8.5.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash -e
+# ----------------------------------------------------------------------------
+# Package          : VIPnytt/SitemapParser
+# Version          : 1.1.4
+# Source repo      : https://github.com/VIPnytt/SitemapParser
+# Tested on        : UBI 8.5
+# Language         : PHP
+# Travis-Check     : True
+# Script License   : Apache License, Version 2 or later
+# Maintainer       : Shalmon Titre <Shalmon.Titre1@ibm.com>
+#
+# Disclaimer       : This script has been tested in root mode on given
+# ==========         platform using the mentioned version of the package.
+#                    It may not work as expected with newer versions of the
+#                    package and/or distribution. In such case, please
+#                    contact "Maintainer" of this script.
+#   
+# ----------------------------------------------------------------------------
+PACKAGE_NAME=SitemapParser
+PACKAGE_VERSION=1.1.4
+PACKAGE_URL=https://github.com/VIPnytt/SitemapParser.git
+
+yum -y update 
+yum module enable php:7.3 -y
+yum install -y nodejs nodejs-devel nodejs-packaging npm python38 python38-devel ncurses git jq curl php php-curl php-dom php-mbstring php-json nodejs make gcc-c++ patch diffutils php-gd php-pecl-zip
+
+php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');" && php composer-setup.php --install-dir=/bin --filename=composer
+yum install php-xdebug -y
+
+OS_NAME=`python3 -c "os_file_data=open('/etc/os-release').readlines();os_info = [i.replace('PRETTY_NAME=','').strip() for i in os_file_data if i.startswith('PRETTY_NAME')];print(os_info[0])"`
+HOME_DIR=`pwd`
+if ! git clone $PACKAGE_URL $PACKAGE_NAME; then
+    	echo "------------------$PACKAGE_NAME:clone_fails---------------------------------------"
+		echo "$PACKAGE_URL $PACKAGE_NAME"
+        echo "$PACKAGE_NAME  |  $PACKAGE_URL |  $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Clone_Fails"
+    	exit 0
+fi
+
+cd $PACKAGE_NAME
+git checkout $PACKAGE_VERSION
+
+sed -i '8 i \ \ \  <filter>' phpunit.xml
```

Comment:
can we create a separate patch file and apply here instead of using so many seds?

Reply:
Updated build script with path.